src/redux/books/books.js

```diff
@@ -0,0 +1,15 @@
+const addBook = () => ({ type: 'ADD' });
+const removeBook = () => ({ type: 'REMOVE' });
```

Comment:
- [ ] Good job on setting up the redux store but there are some issues you need to work on. Please also pass the payload in your actions so your reducer will have data to update the state.
 - You can look at this action for hint `const addBook = (payload) => ({
  type: 'ADD',
  payload,
});`